utils/sane-backends/Makefile

```diff
@@ -125,13 +125,20 @@ $(call Package/sane-backends/Default)
   CATEGORY:=Utilities
   DEPENDS:=+libsane
   TITLE+= (network daemon)
+  USERID:=saned:scanner
 endef
 
 define Package/sane-daemon/description
 $(call Package/sane-backends/Default/description)
 This package contains the SANE daemon.
 endef
 
+# Run hotplug to grant access to existing scanners
+define Package/sane-daemon/postinst
+#!/bin/sh
+udevtrigger
```

Comment:
Is it safe to run udevtrigger here?

Reply:
Or better, should I run udevtrigger to grant SANE access to scanner devices right after an installation or should I let the user decide to run it manually or simply reboot? SANE will not work until either happens.

Reply:
Using udevtrigger should be convenient here, both for devs and users.

My understanding is that at the moment it is invoked at boot time to let those hotplug scripts have a chance to pick up events before uevent watching was present.  The need is sort of the same here.

Reply:
Should this verify we are on the device (e.g. if [ -z "$INST_ROOT" ]: then udevtrigger; fi)?  Otherwise imagebuilder will attempt to run undevtrigger on the host (actually so would the monolithic package build).

Reply:
With ImageBuilder:

```
Finalizing root filesystem...
/home/daniel/Build/openwrt/tarxz-sdk-ib/19.07.0-rc2/ath79/ib-19-ath79/build_dir/target-mips_24kc_musl/root-ath79/usr/lib/opkg/info/sane-daemon.postinst-pkg: line 2: udevtrigger: command not found
postinst script ./usr/lib/opkg/info/sane-daemon.postinst has failed with exit code 127
```
As I suspected.  Doing up a patch now...

Reply:
FYI it applies against 19.07.0-rc2 with not changes required, and I'm hoping to use it with a (mostly 19.07) build (I suspect it won't make the cutoff though).

Reply:
Sorry but this PR was created after 19.07 has branched. IMHO, as this is a feature and not a fix, it should go into the next release.

Reply:
I kind of figured that.  I'll test against master in a while, just to be sure.